stdlib/public/core/VarArgs.swift

```diff
@@ -290,7 +290,9 @@ final public class VaListBuilder {
     // supported vararg type is greater than the alignment of Int.
     // FIXME: this implementation is not portable because
     // alignof differs from the ABI alignment on some architectures
-#if os(watchOS) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
+    // FIXME: This is valid for AAPCS calling convention, FreeBSD/arm
+    // may need this too.
+#if (os(watchOS) || os(Linux)) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
```

Comment:
Are other 32-bits architectures susceptible of this? Wouldn't be best to be !os(iOS) && arch(arm)? Does this enable the tvOS target to use it or iOS includes tvOS?